torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2100,27 +2100,33 @@ def error_inputs_T(self, device, has_ndims_error=False):
                                       r'to reverse their shape is not supported\.'))
 
 
-def sample_inputs_singular_matrix_factors(op_info, device, dtype, requires_grad=False, **kwargs):
+def sample_inputs_singular_matrix_factors(op_info, device, dtype, requires_grad=False):
     """
     This function produces two tensors of shape (*, m, k) and (*, n, k) with k <= min(m, n).
     Their matrix product could be used to generate tensor of shape (*, m, n) of rank k.
     """
 
     make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
-    batches = [(), (0, ), (2, ), (1, 1)]
-    size = [1, 5, 10]
-
+    batches = [(), (2,)]
+    size = [3, 4]
```

Comment:
I would feel more comfortable if reducing the number of sample inputs was left until another PR.

Reply:
the tests were taking forever. It doesn't make sense to have more tests here than in most linalg functions, given how extremely slow it is.

Reply:
I didn't say you couldn't change the test, just split it into another PR so I can see the tests still pass.

Reply:
done